src/190B20.c

```diff
@@ -1072,7 +1072,30 @@ INCLUDE_ASM(s32, "190B20", func_80263268);
 
 INCLUDE_ASM(s32, "190B20", func_80263300);
 
-INCLUDE_ASM(s32, "190B20", btl_are_all_enemies_defeated);
+s32 btl_are_all_enemies_defeated(void) {
+    s32* currentEnemyFlags;
+    s32 i;
+    BattleStatus* bs;
+    s32 enemiesStillAlive;
+    s32 flagEnemyDefeated;
+
+    enemiesStillAlive = FALSE;
+    for (i = 0; i < ARRAY_COUNT(bs->enemyActors); i++) {
+        flagEnemyDefeated = 0x404000;
+        bs = &gBattleStatus;
+        currentEnemyFlags = bs->enemyActors[i];
+
+        // If currentEnemyFlags signify that the enemy isn't dead yet...
+        if (currentEnemyFlags != 0) {
+            if ((*currentEnemyFlags & flagEnemyDefeated) == 0) {
```

Comment:
What's up with the dereference of `currentEnemyFlags` here? It's not a pointer...
Also we write falsey flag checks like `if (!(flags & flag))`